com.unity.multiplayer.mlapi/Runtime/NetworkedVar/NetworkedVar.cs

```diff
@@ -15,15 +15,15 @@ public class NetworkedVar<T> : INetworkedVar
         /// <summary>
         /// Gets or sets Whether or not the variable needs to be delta synced
         /// </summary>
-        public bool isDirty { get; set; }
+        private bool isDirty { get; set; }
```

Comment:
Why is this changed to private?

It's can be quite useful to be able to force the dirty flag, and especially reading it.

Reply:
I disagree making it public and you could manually mark things dirty.
But I agree that it shouldn't be a change in this PR, it needs a separate conversation & PR.

Reply:
Manually marking things dirty is useful for things that doesn't comply with the default comparator. Complex structures as an example.

Reply:
For writing, seemed like the dirty-ification would be generally something you'd want to isolate to code inside NetworkVar.cs.  If arbitrary code can tweak this at an arbitrary time it could create chaos, particularly when we job-ify things.

For reading, there already is an IsDirty() function, though admittedly it checks isDirty AND takes into account some send tickrate stuff.

This isn't essential to this change obvious, was more of an incidental cleanup; @TwoTenPvP if you believe folks will legitimately need to force-dirty stuff I'm happy to revert - pls advise

Reply:
I do know people that do use it. I am fine with changing it, I just don't think it belongs in this PR

Reply:
Ok, reverted to public